sample/web/README.md

```diff
@@ -0,0 +1,156 @@
+# project - Web application
+
+The part contains sample of web UI automation sources and tests
+
+Web tests based on UI interactions using Selenium WebDriver and rest API endpoint from common framework Part:
+
+* Wrapper under WebDriver tool from core layer - [webdriver](../../common/_webdriver_qa_api/web)
+* Rest API endpoints - [api_steps](../api/steps)
+
+## code structure
+
+We support 3 layers structure for code base:
+
+* Builder layer - implementation of Page Object pattern
```

Comment:
Page Object and Endpoint Builder

Reply:
it's related only for web part in this readme, so I think that it will be correct to just call it Builder layer and we specify what it contains in the definition

Reply:
Got it, thx